src/serialization/font_pb2.py

```diff
@@ -0,0 +1,97 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: font.proto
```

Comment:
It looks like this is automatically generated from the `font.proto` file, is that correct? I wonder if we should even commit it then: we usually gitignore auto-generated files (e.g. `.out` or `.o` files). How exactly did this file get generated?